xugrid/regrid/regridder.py

```diff
@@ -235,8 +235,8 @@ def _csr_from_dataset(dataset: xr.Dataset) -> WeightMatrixCSR:
             dataset["__regrid_data"].values,
             dataset["__regrid_indices"].values,
             dataset["__regrid_indptr"].values,
-            dataset["__regrid_n"].values,
-            dataset["__regrid_nnz"].values,
+            dataset["__regrid_n"].values[()],
```

Comment:
why do you need to add "[()]" to only the last two of these ? is it because they are scalar and the rest is array-type? If it's something like that please think about how to make that more obvious to the readers of the code